docs/test-specifications/contract-service/ContractCreateTransaction.md

```diff
@@ -0,0 +1,323 @@
+| 5       | Create a contract with an invalid initcode hex string                                 | `initcode="0xZZ"`                                                                  | SDK/client validation error, transaction not submitted OR network fails with `INVALID_CONTRACT_INITCODE`.             | N                 |
+| 6       | Create a contract with a valid initcode with constructorParameters                    | `initcode=<VALID_INITCODE_HEX>`, `constructorParameters=<VALID_ABI_ENCODED_HEX>`   | Transaction succeeds; constructor runs with provided params (verify via `ContractFunctionResult` on mirror node).     | N                 |
+| 7       | Create a contract with a valid initcode but insufficient gas                          | `initcode=<VALID_INITCODE_HEX>`, `gas="0"`                                         | Transaction fails with `INSUFFICIENT_GAS`.                                                                            | N                 |
+
+---
+
+### **Admin Key**
+- Admin Key is an optional key that grants administrative control over the smart contract.
+
+| Test no | Name                                                                | Input                                                                                                                         | Expected Response                                             | Implemented |
+|---------|---------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------| ----------- |
+| 1       | Create a contract with a valid ED25519 public key as its admin key  | adminKey=<VALID_ED25519_PUBLIC_KEY>, commonTransactionParams.signers=[<CORRESPONDING_VALID_ED25519_PRIVATE_KEY>]              | Transaction succeeds, adminKey reflected in `getContract`.    | N           |
+| 2       | Create a contract with a valid ECDSA key as its admin key           | adminKey=<VALID_ECDSA_PUBLIC_KEY>, commonTransactionParams.signers=[<CORRESPONDING_VALID_ECDSA_PRIVATE_KEY>]                  | Transaction succeeds, adminKey reflected in `getContract`.    | N           |
+| 3       | Create a contract wtith an Invalid key format                       | adminKey=<INVALID_PUBLIC_KEY>                                                                                                 | Fails with SDK error or `INVALID_ADMIN_KEY`.                                                                                  | N           |
+| 4       | Create a contract with no adminKey                                  | —                                                                                                                             | Transaction succeeds, no admin key set. Contract is immutable | N           |
```

Comment:
This looks like a copy of `bytecodeFileId` test no. 2 where there are no provided inputs, but this one succeeds and that one fails.